src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -262,6 +262,11 @@ private static void AddBracketIndentationOperation(List<IndentBlockOperation> li
             // Brackets in list patterns are formatted like blocks, so align close bracket with open bracket
             AddIndentBlockOperation(list, bracketPair.openBracket.GetNextToken(includeZeroWidth: true), bracketPair.closeBracket.GetPreviousToken(includeZeroWidth: true));
 
+            if (IsSwitchExpressionPattern(node))
```

Comment:
this needs a 'why' explanation as to why we don't want to execute teh rest of the code if we hit such a thing.